app/models/project.rb

```diff
@@ -1,5 +1,9 @@
 class Project < ApplicationRecord
   has_many :publish_targets
```

Comment:
@elohanlon Should this also be `dependent: :destroy`?

Reply:
Oh yes, you're right